### ISSUE TITLE
Motions from lightspeed.nvim don't work for certain char arguments

### ISSUE THREAD

**Issue by hallettj** [neovide/neovide#1539]
**Describe the bug**
This bug comes up specifically when using [lightspeed.nvim](https://github.com/ggandor/lightspeed.nvim). I'm reporting it here because the problem occurs in Neovide, but does not occur when running nvim in a terminal.

Lightspeed overrides some movement commands like `f` and `t`. When attempting to use those custom movements to move to certain characters such as `<` instead of the expected behavior I see this error message:

    not found: <80><fc>^B<

I have seen this problem with the characters `<` and `|`. The problem does not occur with `>` or with alphanumeric characters.

Using git bisect I found that this issue was introduced by #1483.

I tried reading the Lightspeed source to come up with a more minimal reproduction, but the Fennel code is a bit much for me to decipher right now. The function that handles 1-character search is [ft.go](https://github.com/ggandor/lightspeed.nvim/blob/main/fnl/lightspeed.fnl#L577).

**To Reproduce**
1. Install [lightspeed.nvim](https://github.com/ggandor/lightspeed.nvim)
2. Open a file with this content: `const x = 1 < 3`
3. With the cursor at the beginning of that line in normal mode, type `yt<`

**Expected behavior**
There should be no error message. The default register should contain the text `const x = 1 `.

**Actual behavior**
This error message is reported upon typing `<`:

    not found: <80><fc>^B<

**Desktop (please complete the following information):**
 - OS: Debian Testing (bookworm)
 - Neovide Versions 346dc05 and later commits in main
 - Neovim Versions NVIM v0.7.2, NVIM v0.8.0-dev-1064-g48e060f64, and NVIM v0.8.0-dev-1092-g89b9eab63 (latest nightly as of Sept 15 2022)

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
https://gist.github.com/hallettj/d5ed3158d1ee9a89a34968502b4cd4ae

**Comment by hallettj** [comment]
I've noticed another issue with the `<` key. When viewing git status with [fugitive](https://github.com/tpope/vim-fugitive) there are bindings for `>` to expand the diff for a file, and `<` collapses the diff. After #1483 pressing `<` in a fugitive buffer puts the editor into operator pending mode instead of collapsing the diff. (But `>` still works as expected.) It's possible there is an interaction with another one of my plugins here. If I get more information I may open another issue.

**Comment by s-cerevisiae** [comment]
Luckily I know some Fennel, and after some tracing I found the minimal reproduction:
```
:echo getcharstr()
```
Then type `<`.

**Comment by s-cerevisiae** [comment]
The fix is now merged. Would you like to test it?

**Comment by hallettj** [comment]
I'm sorry I didn't see your message sooner. Yes, the fix is working nicely, thank you!
